### PR TITLE
fix failing tests due to disc space

### PR DESCRIPTION
**Pull request by emlynjdavies**
This PR modifies the tests to use less disc space, which is more suited to GitHub actions.

### CODE REVIEW

File: pysilcam/tests/test_process.py [hunk 1]
```diff
@@ -8,6 +8,8 @@
 import os
 import unittest
 import pandas as pd
+import shutil
+import glob
```
Comment (tankefugl) [inline pysilcam/tests/test_process.py:12]: F811: redefinition of unused 'glob' from line 7

File: pysilcam/tests/test_process.py [hunk 2]
```diff
@@ -19,6 +21,42 @@
...
+    # make sure export directory is empty
+    shutil.rmtree(os.path.join(data_file, 'export'), ignore_errors=True)
+
+    # call process function
+    silcam_process(conf_file_out, data_file, multiProcess=True, nbImages=5)
+
+    imc_files = glob.glob(os.path.join(data_file, 'export', '*-IMC*'))
+    assert len(imc_files) == 5, 'unexpected number of IMC files'
+
+    seg_files = glob.glob(os.path.join(data_file, 'export', '*-SEG*'))
```
Comment (tankefugl) [inline pysilcam/tests/test_process.py:53]: F841: local variable 'seg_files' is assigned to but never used

File: pysilcam/tests/test_process.py (hunk 1 above)
Comment (tankefugl) [inline pysilcam/tests/test_process.py:12]: F811: redefinition of unused 'glob' from line 7

File: pysilcam/tests/test_process.py [hunk 3]
```diff
@@ -19,6 +20,44 @@
...
+    conf.set('General', 'logfile', os.path.join(ROOTPATH, 'log.log'))
+    conf.set('ExportParticles', 'outputpath', os.path.join(data_file, 'export'))
+    if MODEL_PATH is not None:
+        conf.set('NNClassify', 'model_path', MODEL_PATH)
+    conf_file_hand = open(conf_file_out, 'w')
+    conf.write(conf_file_hand)
+    conf_file_hand.close()
+
+    # make sure export directory is empty
+    shutil.rmtree(os.path.join(data_file, 'export'), ignore_errors=True)
```
Comment (nepstad) [inline pysilcam/tests/test_process.py:44]: Would it be better to use the tempfile library to generate a new randomly named temporary output directory, instead of doing this recursive delete? Such a delete could conceivable go wrong in some edge cases.

Comment (nepstad) [reply pysilcam/tests/test_process.py:44]: For instance with

with tempfile.TemporaryDirectory() as tmpdirname:
    <do stuff>

Comment (emlynjdavies) [reply pysilcam/tests/test_process.py:44]: Nice idea. I am not completely sure the best way to implement the temporary directory in this case, where the directory that needs cleaning is configured within the config file, which is later read from within the processing function:

conf.set('ExportParticles', 'outputpath', os.path.join(data_file, 'export'))
